ui/app/styles/helper-classes/colors.scss

```diff
@@ -72,6 +72,10 @@ select.has-error-border,
 .has-text-info {
   color: $blue-500 !important;
 }
+// same without the !important
+.has-text-primary {
+  color: $blue-500;
+}
```

Comment:
I had originally added this in another PR and was informed about the `has-text-info` class. The use of important however doesn't allow the color to be changed in the case of using on a button for example when disabled so I added it back in.

Reply:
good to know. Thanks. Our CSS still has lots of ordering and important issues.

Reply:
Ya it would be great to remove most if not all instances of `!important`. It would take a good amount of manual verification to make sure that things don't change on each element using the given class but would be worth it in the end.